bazel/external/proxy_wasm_cpp_host.BUILD

```diff
@@ -21,38 +22,40 @@ cc_library(
 cc_library(
     name = "lib",
     # Note that the select cannot appear in the glob.
-    srcs = envoy_select_wasm_all_v8_wavm_none(
-        glob(
-            [
-                "src/**/*.h",
-                "src/**/*.cc",
-            ],
-        ),
-        glob(
-            [
-                "src/**/*.h",
-                "src/**/*.cc",
-            ],
-            exclude = ["src/wavm/*"],
-        ),
-        glob(
-            [
-                "src/**/*.h",
-                "src/**/*.cc",
-            ],
-            exclude = ["src/v8/*"],
-        ),
-        glob(
-            [
-                "src/**/*.h",
-                "src/**/*.cc",
-            ],
-            exclude = [
-                "src/wavm/*",
-                "src/v8/*",
-            ],
-        ),
-    ),
+    srcs = envoy_select_wasm([
+        "include/proxy-wasm/context.h",
+        "include/proxy-wasm/context_interface.h",
+        "include/proxy-wasm/exports.h",
+        "include/proxy-wasm/null.h",
+        "include/proxy-wasm/null_plugin.h",
+        "include/proxy-wasm/null_vm.h",
+        "include/proxy-wasm/null_vm_plugin.h",
+        "include/proxy-wasm/wasm.h",
+        "include/proxy-wasm/wasm_api_impl.h",
+        "include/proxy-wasm/wasm_vm.h",
+        "include/proxy-wasm/word.h",
+        "src/context.cc",
+        "src/exports.cc",
+        "src/foreign.cc",
+        "src/wasm.cc",
+        "src/null/null.cc",
+        "src/null/null_plugin.cc",
+        "src/null/null_vm.cc",
+        "src/third_party/base64.cc",
+        "src/third_party/base64.h",
+        "src/third_party/picosha2.h",
```

Comment:
I think you can still do glob, including extra header file shouldn't result symbol conflict.

Reply:
done: https://github.com/envoyproxy/envoy/pull/13932/commits/b00c82a6198f2242dc98b7606ae754b56d1fed61